packages/contracts-bedrock/test/kontrol/proofs/L1CrossDomainMessenger.k.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import { DeploymentSummary } from "./utils/DeploymentSummary.sol";
+import { KontrolUtils } from "./utils/KontrolUtils.sol";
+import {
+    IL1CrossDomainMessenger as L1CrossDomainMessenger,
+    ISuperchainConfig as SuperchainConfig
+} from "./interfaces/KontrolInterfaces.sol";
+
+contract L1CrossDomainMessengerKontrol is DeploymentSummary, KontrolUtils {
+    L1CrossDomainMessenger l1CrossDomainMessenger;
+    SuperchainConfig superchainConfig;
+
+    /// @dev Inlined setUp function for faster Kontrol performance
+    ///      Tracking issue: https://github.com/runtimeverification/kontrol/issues/282
```

Comment:
Do you have any more details than what's in this issue? Mainly wondering if this is another potential performance improvement in the near future, or if this is a low priority issue

Reply:
My current understanding is that there is some extra logic being added to the execution of the `setUp` function, for instance, to prevent/check for branching on it, and this slows down its execution when compared to just inlining it.
Since the logic currently performed by the `setUp` is just assigning two variables, the overhead for inlining it is not big, but it could be a small improvement once we figure out how to reduce this `setUp` overhead